src/main/kotlin/no/nav/syfo/personstatus/PersonoversiktStatusService.kt

```diff
@@ -23,6 +23,7 @@ class PersonoversiktStatusService(
         val personListe = database.hentUbehandledePersonerTilknyttetEnhet(
             enhet = enhet,
         )
+        val today = LocalDate.now()
```

Comment:
Trenger dagens dato i denne sammenhengen, så instansierer den opp en gang og ikke for hver rad som skal vurderes (siden det kan være mange).